client/dashboard/components/notice/types.ts

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 type NoticeDensity = 'low' | 'medium' | 'high';
 
-export type NoticeVariant = 'warning' | 'success' | 'error' | 'info';
```

Comment:
@lucasmendes-design I've gone ahead and added a "promote" variant of the notice which displays the star icon. It uses the same colours as the "info" variant.

I wonder what the difference here and why this notice wouldn't have shown a regular (i) logo, since this seems semantically just like an info notice 🤔

I've seen the star icon used in various places, but are we confident enough about the notice style that we want to create the new variant?

Reply:
It looks like a custom icon will use the "default" intent in ARC-690. Since we can't say semantically what it means anymore. But for our dashboard component here I'll just stick with using the `info` variant for this notice. Hopefully we'll be able to drop in their component eventually, and we'll just have to conform to how it's been implemented there.

Reply:
It seems it's still in discussion, for instance, there was this idea as well: "non-default intents show an associated icon (that can be changed / removed);"